native/build.gradle

```diff
@@ -90,7 +90,7 @@ tasks.withType(Checkstyle) {
 }
 
 spotbugsMain.finalizedBy validateSpotbugs
-checkstyleMain.dependsOn(":build-config:checkstyle:downloadMultipleFiles")
+checkstyleMain.dependsOn(":checkstyle:downloadMultipleFiles")
```

Comment:
Shall we change this to `downloadCheckstyleRuleFiles`?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-xslt/pull/196/commits/4cbc6bfe505c860ef0887ace3aa3f45183ca57f5